website/src/pages/en/subgraphs/querying/graphql-api.mdx

```diff
@@ -24,15 +24,15 @@ Explore the GraphQL Query API for interacting with Subgraphs on The Graph Networ
   - Read-only operations (no mutations)
   - Defines entity interfaces and derived fields
 
-## Queries with GraphQL
+## Query Structure
 
-In the Subgraph schema, types called `Entities`. For each `Entity` type, `entity` and `entities` fields will be generated on the top-level `Query` type.
+GraphQL queries in The Graph target entities defined in the Subgraph schema. Each `Entity` type generates corresponding `entity` and `entities` fields on the root `Query` type.
 
-### Example Queries
+Note: The `query` keyword is not required at the top level of GraphQL queries.
 
-> Note: `query` does not need to be included at the top of the `graphql` query when using The Graph.
+### Single Entity Queries
 
-Query for a single `Token` entity defined in your schema:
+Query format for a single `Token` entity:
```

Comment:
I find the addition of the word "format" in this context to be more confusing than the previous phrasing.

Reply:
Thanks for flagging - agreed.